index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <title>Portfolio David Tamayo</title>
+</head>
+<body>
+    <div class="mainpage">
+        <header class="toolbar">
+            <a class="welcome">Welcome</a>
+            <nav><button class="vector">☰</button></nav>
+        </header>
+        <div class="headline">
+            <h1 class="black">Hey there. I'm David</h1>
+            <p class="green">I'm a software developer</p>
+        </div>
+        <h3 class="content"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
+        <footer class="social">
+            <ul class="sociallinks">
+                <li><a href="https://twitter.com/David5TM"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="www.linkedin.com/in/ing-david-tamayo"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href=""><i class="fa-solid fa-m"></i></a></li>
+                <li><a href="https://github.com/IngDavidTM"><i class="fa-brands fa-github"></i></a></li>
```

Comment:
- [ ] Please kindly fix the background of the page as mentioned by the previous reviewer to make your project look professional. you got this 👍 

your design:
![image](https://user-images.githubusercontent.com/90524466/185455081-d7d625f6-aa91-4bda-9e75-cb9274dd49fb.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185455548-e5fdf7fd-fc71-436f-a986-3929e6603b3e.png)

Reply:
Hi @mugishasam123 as I mentioned before on my computer the design seems normally. I've already adjusted my design for a Minimum mobile screen size but a linter error showed up :(

Reply:
@mugishasam123 Could you tell me what you use to run the project as a mobile? Because in Chrome the design does appear. Please

Reply:
![Captura de pantalla de 2022-08-18 09-15-43](https://user-images.githubusercontent.com/105117832/185459765-0831a295-4dea-4839-9863-c16e94e31efa.png)